.github/workflows/integration_test.yaml

```diff
@@ -1,6 +1,16 @@
 name: Integration Tests
 
-on: [push, pull_request]
+on:
+  pull_request:
+    branches:
+      - 'main'
+      - 'next'
+  push:
+    branches:
+      - 'main'
+      - 'next'
+    tags:
+      - 'v*.*.*'
```

Comment:
I did this intentionally because it was double running the CI on PRs (one for the branch, one for the PR) I would like to run with this for a while, as the CI runs are somewhat long and aren't zero cost.